invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagement.js

```diff
@@ -126,6 +126,7 @@ export class CommunitiesManagement extends Component {
               error={error}
               loading={loading}
               maxDisplayedCommunities={MAX_COMMUNITIES}
+              branded={record.parent?.communities?.default}
```

Comment:
question 1: only one community can have the `verified` icon?

question 2: why is the `default` community the one considered `branded`?

Reply:
About (1) - just as info, records already have a `verified` field but it's related to content moderation.